docs/reference/setup/restart-cluster.asciidoc

```diff
@@ -1,9 +1,12 @@
 [[restart-cluster]]
-== Restart a cluster
+== Full-cluster restart and rolling restart
```

Comment:
This isn't blocking, but I wonder if these might work better as separate sections that reuse tagged content regions.